types/office-js-preview/index.d.ts

```diff
@@ -330,6 +330,10 @@ declare namespace Office {
      * @param useShortNamespace True to use the shortcut alias; otherwise false to disable it. The default is true.
      */
     function useShortNamespace(useShortNamespace: boolean): void;
+    /**
+     * Represents the actions.
```

Comment:
The Actions interface really doesn't represent actions. It just provides a mapping service. 
```suggestion
     * Provides a method for associating action names with functions that carry out an action..
```